.github/workflows/ci.yml

```diff
@@ -100,19 +123,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test-category:
-          - integration-bft
-          - integration-mvp
-          - integration-ghost
-          - integration-network
-          - integration-epochs
-          - integration-access
-          - integration-collection
-          - integration-consensus
-          - integration-execution
-          - integration-verification
-    env:
-      TEST_CATEGORY: ${{ matrix.test-category }}
+        make:
+          - make -C integration access-tests
+          - make -C integration bft-tests
+          - make -C integration collection-tests
+          - make -C integration consensus-tests
+          - make -C integration epochs-tests
+          - make -C integration execution-tests
+          - make -C integration ghost-tests
+          - make -C integration mvp-tests
+          - make -C integration network-tests
+          - make -C integration verification-tests
```

Comment:
do we need that change? maybe we can leave it as it?

Reply:
This is very important - it eliminates the need to use `run-tests.sh` which would hide failures and make it a lot harder to debug. I want to completely remove `run-tests.sh` from unit and integration test runs here in CI and eventually in `flaky-test-monitor.yml` CI workflow.